webapi/Skills/ChatSkills/ChatSkill.cs

```diff
@@ -371,20 +380,24 @@ private async Task<ChatMessage> GetChatResponseAsync(string chatId, string userI
 
             // Save a new response to the chat history with the proposed plan content
             return await this.SaveNewResponseAsync(
-                JsonSerializer.Serialize<ProposedPlan>(proposedPlan), prompt, chatId, userId,
+                JsonSerializer.Serialize<ProposedPlan>(proposedPlan),
+                prompt,
+                chatId,
+                userId,
                 // TODO: [Issue #2106] Accommodate plan token usage differently
-                this.GetTokenUsagesAsync(chatContext)
+                this.GetTokenUsages(chatContext),
```

Comment:
Maybe the following is better:

```suggestion
                await this.GetTokenUsagesAsync(chatContext),
```

Reply:
The method isn't asynchronous tho, this was a slip on my part when implementing it
![image](https://github.com/microsoft/chat-copilot/assets/125500434/879206cb-c71d-48ce-b9bd-482a7d51c6d6)

Reply:
Never mind, then!